webhooks/payments.go

```diff
@@ -28,6 +28,7 @@ type PaymentNotification struct {
 type Transaction struct {
 	XMLName           xml.Name         `xml:"transaction"`
 	UUID              string           `xml:"id"`
+	InvoiceID         string           `xml:"invoice_id"`
```

Comment:
This is the thing we need in this struct in order for https://github.com/splice/api/pull/8520 to be able to add InvoiceID